Profiles/ext-ConditionStatusDate.xml

```diff
@@ -15,8 +15,8 @@
       <use value="work" />
     </telecom>
   </contact>
-  <description value="An extension to provide the moment from when the data regarding the condition apply. It may be a vague moment.&#xD;&#xA;&#xA;The ConditionStatusDate has some overlap with other concepts in this data set.&#xA;* For initial Hypersensitivities and Reactions with ConditionPresence 'Known present', the ConditionStatusDate is always equal to the DiagnosticInsightDate.&#xA;* For updated Hypersensitivities and Reactions with ConditionPresence 'Known present' the ConditionStatusDate is in most cases equal to the RegistrationDateTime.&#xA;* For Hypersensitivities and Reactions with ConditionPresence 'Not present anymore' or 'Known absent' the ConditionStatusDate corresponds to the abatement date of the condition." />
-  <purpose value="This extension represents the ConditionStatusDate concept for implementations following the information standard [CiO](https://informatiestandaarden.nictiz.nl/wiki/Landingspagina_Contra-indicaties_en_Overgevoeligheden)." />
+  <description value="The moment from when the data regarding the condition apply. It may be a vague moment.&#xD;&#xA;&#xD;&#xA;The StatusDate has some overlap with other concepts in this data set.&#xA;* For initial Hypersensitivities and Reactions, the StatusDate is always equal to the DiagnosticInsightDate.&#xA;* For updated Hypersensitivities and Reactions, the StatusDate is in most cases equal to the RegistrationDateTime." />
```

Comment:
In the previous iteration of the data set there were concepts ConditionPresence and ConditionStartDateTime, which seem to be more or less replaced by the PeriodPresent concept. In the description on the left there was more guidance with respect to active and inactive Conditions. Is there a reason this has been removed in the new description?
(ConditionPresence 'Known present' corresponds with a Condition that has no PeriodPresent.TimeInterval.endDateTime, while ConditionPresence 'Not present anymore' or 'Known absent' corresponds with a Condition that has PeriodPresent.TimeInterval.endDateTime specified.)